documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -433,7 +433,15 @@ For more information about the `livenessProbe` and `readinessProbe` options, see
 == `metricsConfig`
 
 Use the `metricsConfig` property to enable and configure Prometheus metrics.
+Strimzi provides support for Prometheus JMX Exporter and Strimzi Metrics Reporter. Only one of these can be selected at any given time.
 
+When metrics are enabled, they are exposed on port 9404.
+
+When the `metricsConfig` (or deprecated `metrics`) property is not defined in the resource, the Prometheus metrics are disabled.
```

Comment:
Maybe we don't need the deprecated mention in brackets anymore?

Reply:
Removed